// @hubspot/ui-extensions@0.10.1/dist/experimental/index.js
import { createRemoteReactComponent } from '@remote-ui/react';
export { useCrmProperties } from '../crm/hooks/useCrmProperties';
export { useAssociations } from '../crm/hooks/useAssociations';
/** @experimental This component is experimental. Avoid using it in production due to potential breaking changes. Your feedback is valuable for improvements. Stay tuned for updates. */
const Iframe = createRemoteReactComponent('Iframe');
/** @experimental This component is experimental. Avoid using it in production due to potential breaking changes. Your feedback is valuable for improvements. Stay tuned for updates. */
const MediaObject = createRemoteReactComponent('MediaObject', {
    fragmentProps: ['itemRight', 'itemLeft'],
});
/** @experimental This component is experimental. Avoid using it in production due to potential breaking changes. Your feedback is valuable for improvements. Stay tuned for updates. */
const Stack2 = createRemoteReactComponent('Stack2');
/** @experimental This component is experimental. Avoid using it in production due to potential breaking changes. Your feedback is valuable for improvements. Stay tuned for updates. */
const Center = createRemoteReactComponent('Center');
/** @experimental This component is experimental. Avoid using it in production due to potential breaking changes. Your feedback is valuable for improvements. Stay tuned for updates. */
const Grid = createRemoteReactComponent('Grid');
/** @experimental This component is experimental. Avoid using it in production due to potential breaking changes. Your feedback is valuable for improvements. Stay tuned for updates. */
const GridItem = createRemoteReactComponent('GridItem');
const SettingsView = createRemoteReactComponent('SettingsView');
/**
 * The `ExpandableText` component renders a text that can be expanded or collapsed based on a maximum height.
 *
 * **Links:**
 *
 * - {@link https://developers.hubspot.com/docs/reference/ui-components/standard-components/expandable-text ExpandableText Docs}
 */
const ExpandableText = createRemoteReactComponent('ExpandableText');
/**
 * The `Popover` component renders a popover overlay that can contain other components.
 *
 * **Links:**
 *
 * - {@link https://developers.hubspot.com/docs/reference/ui-components/standard-components/popover Popover Docs}
 */
const Popover = createRemoteReactComponent('Popover');
const FileInput = createRemoteReactComponent('FileInput');
export { Iframe, MediaObject, Stack2, Center, GridItem, Grid, SettingsView, ExpandableText, Popover, FileInput, };
